{"name": "测试数据：【 wintest，123456，True，0，登录成功】", "status": "failed", "statusDetails": {"message": "AssertionError: 接口返回码是 【 1003 】, 返回信息：用户名不存在！！！ \nassert False == True\n +  where False = <core.result_base.ResultBase object at 0x7fe8a1f227f0>.success", "trace": "self = <testcases.api_test.test_03_login.TestUserLogin object at 0x7fe8a1fc3da0>\nusername = 'wintest', password = '123456', except_result = True, except_code = 0\nexcept_msg = '登录成功'\n\n    @allure.story(\"用例--登录用户\")\n    @allure.description(\"该用例是针对获取用户登录接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{password}，{except_result}，{except_code}，{except_msg}】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, password, except_result, except_code, except_msg\",\n                             api_data[\"test_login_user\"])\n    def test_login_user(self, username, password, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        result = login_user(username, password)\n        step_1(username)\n>       assert result.success == except_result, result.error\nE       AssertionError: 接口返回码是 【 1003 】, 返回信息：用户名不存在！！！ \nE       assert False == True\nE        +  where False = <core.result_base.ResultBase object at 0x7fe8a1f227f0>.success\n\ntestcases/api_test/test_03_login.py:30: AssertionError"}, "description": "该用例是针对获取用户登录接口的测试", "steps": [{"name": "步骤1 ==>> 登录用户", "status": "passed", "parameters": [{"name": "username", "value": "'wintest'"}], "start": 1751426292421, "stop": 1751426292421}], "attachments": [{"name": "log", "source": "1515336a-808f-4b1a-9c3a-ce438f32385f-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "3c28d460-adac-47a9-b710-59fed7613bfb-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'wintest'"}, {"name": "password", "value": "'123456'"}, {"name": "except_result", "value": "True"}, {"name": "except_code", "value": "0"}, {"name": "except_msg", "value": "'登录成功'"}], "start": 1751426292398, "stop": 1751426292422, "uuid": "7c8ed366-02be-4c03-b41b-28d2d1c53e11", "historyId": "4ab02ebad4baabe28d2fb7352ad2343a", "testCaseId": "7272db244dabef390096479ae3196642", "fullName": "testcases.api_test.test_03_login.TestUserLogin#test_login_user", "labels": [{"name": "epic", "value": "针对单个接口的测试"}, {"name": "severity", "value": "normal"}, {"name": "story", "value": "用例--登录用户"}, {"name": "feature", "value": "用户登录模块"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_03_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "iZ2ze5i9ndpg9iyd748go2Z"}, {"name": "thread", "value": "83277-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_03_login"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}